### PR TITLE
Fixed extraction message type

### PR DESCRIPTION
**Pull request by Guillar1**

### CODE REVIEW

File: src/reporting/interfaces/ExtractionProcess.ts [hunk 1]
```diff
@@ -151,7 +151,7 @@ export interface ExtractionLog {
    * @type {string}
    * @memberof ExtractionLog
    */
-  message?: string;
+  message: string | null;
```
Comment (MindaugasButkus) [inline src/reporting/interfaces/ExtractionProcess.ts:154]: This is a fix, but it might break applications that use this client. Minor version bump?

Comment (Guillar1) [reply src/reporting/interfaces/ExtractionProcess.ts:154]: Makes sense. Ibumped the minor version.

Comment (MindaugasButkus) [reply src/reporting/interfaces/ExtractionProcess.ts:154]: Run `npm install` to update the version in lock file.

Comment (Guillar1) [reply src/reporting/interfaces/ExtractionProcess.ts:154]: true that.

**Review by MindaugasButkus — APPROVED** [review]

**Pull request merged**